src/utils/getIsReactionEnabled.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * This function helps consider the every condition
+ * related to enabling emoji reaction feature.
+ */
+
+export interface IsReactionEnabledProps {
+  appLevel?: boolean;
+  isBroadcast?: boolean;
+  isSuper?: boolean;
+  globalLevel?: boolean;
+  moduleLevel?: boolean | null;
```

Comment:
Is there any specific reason we allow null for moduleLevel only?

Reply:
Because I set the default value with `null`.
Do you think it's better to keep it undefined? then I will change

Reply:
I thought it'd be better if we can set its default value \w false or true as other fields do.